src/integrationTest/java/io/aiven/kafka/connect/jdbc/AbstractIT.java

```diff
@@ -0,0 +1,128 @@
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+
+import org.apache.avro.generic.GenericRecord;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Testcontainers
+public abstract class AbstractIT {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractIT.class);
+    protected static final String TEST_TOPIC_NAME = "test_topic";
+    private static final String DEFAULT_KAFKA_TAG = "5.4.3";
+    private static final DockerImageName DEFAULT_IMAGE_NAME =
+        DockerImageName.parse("confluentinc/cp-kafka")
+            .withTag(DEFAULT_KAFKA_TAG);
+    protected static KafkaProducer<String, GenericRecord> producer;
+    @Container
+    protected KafkaContainer kafkaContainer = new KafkaContainer(DEFAULT_IMAGE_NAME)
+        .withNetwork(Network.newNetwork())
+        .withEnv("KAFKA_AUTO_CREATE_TOPICS_ENABLE", "false");
+
+    @Container
+    protected SchemaRegistryContainer schemaRegistryContainer =
+        new SchemaRegistryContainer(kafkaContainer);
+
+    protected ConnectRunner connectRunner;
+
+    @BeforeEach
+    void startKafka() throws Exception {
+        LOGGER.info("Configure Kafka connect plugins");
+        setupKafka();
+        final Path pluginDir = setupPluginDir();
+        setupKafkaConnect(pluginDir);
+        producer = createProducer();
+    }
+
+    private static Path setupPluginDir() throws Exception {
+        final Path testDir = Files.createTempDirectory("aiven-kafka-connect-jdbc-test-");
+        final Path distFile = Paths.get(System.getProperty("integration-test.distribution.file.path"));
+        assert Files.exists(distFile);
+
+        final var pluginDir = Paths.get(testDir.toString(), "plugins/aiven-kafka-connect-jdbc/");
+        Files.createDirectories(pluginDir);
+
+        final String cmd = String.format("tar -xf %s --strip-components=1 -C %s", distFile, pluginDir);
+        final Process p = Runtime.getRuntime().exec(cmd);
+        assert p.waitFor() == 0;
+        return pluginDir;
+    }
+
+    private void setupKafka() throws Exception {
+        LOGGER.info("Setup Kafka");
+        try (final AdminClient adminClient = createAdminClient()) {
+            LOGGER.info("Create topic {}", TEST_TOPIC_NAME);
+            final NewTopic newTopic = new NewTopic(TEST_TOPIC_NAME, 4, (short) 1);
```

Comment:
This should probably be moved to the subclass. Currently the only subclass uses this topic. If there was another test, it could "share" the topic, but create it's own.

This could be refactored so that there'd be an `abstract` method `getTopic()` and the subclass would return the topic name. That way, it'd be possible to easily add another IT that would have its own topic name.